packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -56,15 +56,24 @@ export class FASTAccordion extends FASTElement {
         this.$emit("change", this.activeid);
     };
 
-    private findExpandedItem(): FASTAccordionItem | null {
+    private findActiveItem(): FASTAccordionItem | null {
         for (let item: number = 0; item < this.accordionItems.length; item++) {
-            if (this.accordionItems[item].getAttribute("expanded") === "true") {
+            if (!this.accordionItems[item].hasAttribute("disabled")) {
                 return this.accordionItems[item] as FASTAccordionItem;
             }
         }
         return null;
     }
 
+    private findExpandedItem(): FASTAccordionItem | null {
+        for (let item: number = 0; item < this.accordionItems.length; item++) {
+            if (this.accordionItems[item].hasAttribute("expanded")) {
+                return this.accordionItems[item] as FASTAccordionItem;
+            }
+        }
+        return this.findActiveItem();
+    }
+
```

Comment:
I think the change is likely more complex than the refactor that's been made here. I'm going to push a PR up to see if it'll solve the issue here. We'll need to observe all items to see if they have disabled attribute changes which will require some additional logic and a bit of refactoring to how we manage "enabled" items.